### PR TITLE
test: add e2e test scaffold

### PR DESCRIPTION
**Pull request by tokers**
Closing https://github.com/apache/apisix-ingress-controller/issues/74.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,6 +10,7 @@ require (
 	github.com/gxthrj/apisix-types v0.1.0
 	github.com/gxthrj/seven v0.1.9
 	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
+	github.com/onsi/ginkgo v1.11.0
```
Comment (membphis) [inline go.mod:13]: seems wrong. only test cases need this library

Comment (tokers) [reply go.mod:13]: Since we only use on go module in this project, all dependencies include the tests will be recorded in go.mod.

Comment (membphis) [reply go.mod:13]: The `go.sum` and `go.mod` in the test can be maintained separately.

image

Comment (tokers) [reply go.mod:13]: OK.

Comment (tokers) [reply go.mod:13]: Resolve.

**Review by gxthrj — APPROVED** [review]

**Pull request merged**